src/validation.cpp

```diff
@@ -2929,9 +2929,79 @@ bool CChainState::ConnectBlock(const CBlock& block, CValidationState& state, CBl
                     for (const auto& col : collaterals.balances) {
                         cache.SubVaultCollateral(vaultId, {col.first, col.second});
                     }
+                    constexpr const uint64_t batchThreshold = 10000 * COIN; // 10k USD
```

Comment:
With `constexpr` why still need `const`?

Reply:
Actually it can be skipped, `constexpr` is evaluated at compile time, so when it's not `const` it can be changed compile time.